gateway/internal/stats/stats.go

```diff
@@ -100,6 +105,7 @@ func (ss *SourceStat) Report(s stats.Stats) {
 	}
 
 	s.NewTaggedStat("gateway.write_key_requests", stats.CountType, tags).Count(ss.requests.total)
+	s.NewTaggedStat("gateway.write_key_bot_requests", stats.CountType, tags).Count(ss.requests.bot)
```

Comment:
what if we keep the same metric name, but add a bot label? 
```suggestion
	s.NewTaggedStat("gateway.write_key_requests", stats.CountType, tags + {bot=true}).Count(ss.requests.bot)
```

Reply:
We would need to change in Grafana as well then, right?